fix(widget): always force show window on activate event

sudden6 opened the pull request:
fixes #5459

I tested this on KDE, where it still works without any regression.

- [ ] Commits follow our [git commit guidelines](https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#git-commit-guidelines)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qtox/qtox/5466)
<!-- Reviewable:end -->

sudden6 committed:
fix(widget): always force show window on activate event

fixes #5459

sudden6 commented:
@anthonybilinski can you review?

anthonybilinski reviewed (commented):
Do you have a root cause on this? I guess what's happening is that on FreeBSD, the window is active even though it's hidden, skipping the forceShow which both activates and shows the window, but on other OSes, the window isn't considered active when hidden, hitting our check that then both shows and activates it.  I don't see anything in docs or online indicating this would be different for FreeBSD, but if my understand sounds reasonable, <img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/5466#-:-LVrHUlb1o5Nb2INPQpo:b-ssqpes)* status: :shipit: complete! 1 of 1 LGTMs obtained



<!-- Sent from Reviewable.io -->

sudden6 commented:
Yeah, I guessed that this condition could behave differently on different window systems from the descriptions in the linked bug report. After I removed the condition I saw no regression on a system that worked before, so I removed it unconditionally.

anthonybilinski commented:
@sudden6 please rebase onto master to fix CI + merge when you can. Might as well get this into v1.17.0.
